packages/nextjs/components/Activities.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import { ActivitiesItems, TActivityItemProps, TWinnerItemProps, WinnerItem } from "./ActivitiesItem";
+
+export type TActivitiesProps = {
+  rolls: TActivityItemProps[];
+  winners: TWinnerItemProps[];
```

Comment:
Nitpick: I think `winnerRolls` is better. And types are better to change, too. To something like
```
export type TActivitiesProps = {
  rolls: Roll[];
  winnerRolls: Roll[]; // or WinnerRoll[]
}
```

Reply:
There are two different times of even Winner Event and Roll Event 

`event Roll(address indexed player, uint256 amount, uint256 roll);`

`event Winner(address winner, uint256 amount);`

That what i base the types  on